idd/Energy+.idd.in

```diff
@@ -23824,6 +23824,40 @@ AirflowNetwork:MultiZone:WindPressureCoefficientValues,
       \units dimensionless
       \note Enter the WPC Value corresponding to the 36th wind direction.
 
+AirflowNetwork:ZoneControl:PressureController,
+	 \memo Define the Pressures control settings for a zone or list of zones.
+	 \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
+	 \memo to all the zones in the ZoneList.
+ A1 , \field Name
+      \required-field
+      \reference AirflowNetworkZoneControlPressureControllerNames
+ A2 , \field Zone Name
```

Comment:
Will this really work for a ZoneList?  Isn't the control object specific to a single zone?